src/sentry/tagstore/base.py

```diff
@@ -201,15 +208,14 @@ def get_tag_value_paginator_for_projects(
 
     def get_group_tag_value_iter(
         self,
-        group,
-        environment_ids,
-        key,
-        callbacks=(),
```

Comment:
the only user of `callbacks=` was the thing doing the unsound action so I was able to clean this up too!